bolt.fun/site-plan.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+nav_order: 3
+title: Site Plan
+secondary_nav_title: Site Plan
+description:
+permalink: /site-plan/
+main_nav: true
+secondary_nav: true
+main_classes: -no-top-padding
+
+---
+
+# Site Plan
+
+The following is a high-level structure of how we would like to see this mini-site evolve. It gives some idea of our research plans and progress with that research.
+
+---
+
+- **Home**
+
+  _Introduce the idea behind bolt.fun and give high-level overview of content contained_
+
+    - [**Invoices**](/guide/invoices)
+
+      _Introduce basic concept of what a standard invoice is and how it works_
+
+        - Invoice elements
+
+            - HTLCs
+
+            - preimage
+
+            - tlv field
+
+        - [Variations of invoices _(to populate...)_](/guide/invoices/variations/)
+
+            - [The HODL Invoice](/guide/invoices/variations/hodl-invoice)
+
+            - [Keysend](/guide/invoices/variations/keysend)
+
+            - [Multi-part Payments](/guide/invoices/variations/mpp)
+
+            - [Submarine swaps](/guide/invoices/variations/submarine-swap)
```

Comment:
this should be its own section.

Reply:
How you thinking (like which other section)?

I put it here because it's a manipulation of preimages and invoices to accomplish a fund transfer, and leads off from some of the other examples in this section (like HODL invoices).

Reply:
The rationale is that it combines lighting invoice primitives as well as on-chain transaction primitives to create an abstraction.

---

the primitives 👇 

#### Invoices
- Standard Invoice
- HODL Invoice
- Keysend
- Multi-part Payments
- Lightning Offers

#### LSP
- introduction
- opening channels
- capacity
- watch towers?
  - re: mobile limitations

👇  use cases, and interesting combinations

#### Opening Channels
- introduction
- ... [tubro](https://medium.com/breez-technology/lightning-payment-channels-a-flavor-for-every-palette-53cb975907ea)
- ...?

#### Swaps
- introduction
- ...from on-chain
  - ...channel capacity
  - ...toxic change
- ...to on-chain

#### Storage
- ... 🤷 
- [receipts](https://github.com/peakshift/bitcoin-ux/blob/ed9d3689d187487a4d2c4a20904bed86e130dd7c/payments/receipts.md)

Reply:
Ahh ok yea this makes sense. Ok I'll re-org that to match this sort of layout